tinymce-basics/useful_styles_for_content_css.html

```diff
@@ -124,7 +124,7 @@ <h1>Useful content css</h1>
         <p>The default content css is very basic and unopinionated to allow you as a developer to integrate the styles you want in order to get a good WYSIWYG writing environment.</p>
         <p>That said, here are some tecniques we often implement ourselves that improves the editing experience in TinyMCE</p>
         <p><em>Inspect the source code to see the actual css used</em></p>
-        <h2>Improve hr clickability</h2>
```

Comment:
The purpose of this sentence is to show how to improve the clickability of the `hr` element, and removing the word "hr" makes the sentence not make any sense